sd-inference/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM gooseai/torch-base:6cfdc11
+
+RUN apt-get install -y cuda-nvcc-11-3 cuda-nvml-dev-11-3 libcurand-dev-11-3 \
+                       libcublas-dev-11-3 libcusparse-dev-11-3 \
+                       libcusolver-dev-11-3 cuda-nvprof-11-3 \
+                       ninja-build git && \
+    apt-get clean
+
+RUN mkdir /app
+WORKDIR /app
+
+RUN git clone -b amercurio/sd-finetuner https://github.com/coreweave/kubernetes-cloud && \
```

Comment:
Let's get this PR merged into master so this can point to master.

Reply:
Yep, PR is merged now so I'll update the repos to point to master instead of PR